README.md

```diff
@@ -6,11 +6,7 @@ The Greek word for _swift_ and the ship used by Jason, son of Aeson, of the
 Argonauts. Aeson is the JSON parsing library in Haskell that inspired Argo,
 much like Aeson inspired his son Jason.
 
-NOTE: The master branch of Argo is pushing ahead with Swift 1.2 support.
-Support for Swift 1.1 can be found on branch [swift-1.1] and in the 0.3.x
-versions of tags/releases.
-
-[swift-1.1]: https://github.com/thoughtbot/Argo/tree/swift-1.1
+NOTE: For Swift 1.1 support, use the 0.3.x versions of tags/releases.
```

Comment:
"use the versions tagged with 0.3.x, which can read about in the [releases]", and link releases to https://github.com/thoughtbot/Argo/releases